scripts/monorepo/find-and-publish-all-bumped-packages.js

```diff
@@ -103,13 +103,13 @@ const findAndPublishAllBumpedPackages = () => {
 
       const npmOTPFlag = NPM_CONFIG_OTP ? `--otp ${NPM_CONFIG_OTP}` : '';
 
-      const {stderr} = spawnSync('npm', ['publish', `${npmOTPFlag}`], {
+      const {status} = spawnSync('npm', ['publish', `${npmOTPFlag}`], {
         cwd: packageAbsolutePath,
         shell: true,
         stdio: 'pipe',
         encoding: 'utf-8',
       });
-      if (stderr) {
+      if (status !== 0) {
```

Comment:
can we maybe get back both stderr and status and if status is != 0 print both status and stderr? To get better understanding of what's failing 👍

Reply:
> can we maybe get back both stderr and status and if status is != 0 print both status and stderr? To get better understanding of what's failing 👍

Yeah, somehow missed that, will get it back